man/systemd.service.xml

```diff
@@ -1140,7 +1140,11 @@
         shall be considered preferred or less preferred candidates for process termination by the Linux OOM
         killer logic. See
         <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
-        details.</para></listitem>
+        details.</para>
+
+        <para>This setting also applies to <command>systemd-oomd</command>, similar to kernel OOM kills
+        this setting determines the state of the service after <command>systemd-oomd</command> kills a cgroup associated
+        with the service.</para></listitem>
```

Comment:
@nishalkulkarni is the "OOMScoreAdjust=" setting recognized by sd-oomd as of this PR (I cannot find anything in code that would relate to this), or has this been the case for previous releases as well, and you just updated the documentation/man page?

Reply:
Never mind. This is only about the `OOMPolicy=` setting (which is part of the PR), I was confused.

Reply:
ah yeah, sorry, its positioning might confuse people... don't think it's worth a new pull request though